src/backend/curses/pan.rs

```diff
@@ -0,0 +1,221 @@
+        if let Some(ev) = self.window.getch() {
+            match ev {
+                pancurses::Input::Character(c) => Event::Char(c),
+                pancurses::Input::Unknown(i) => Event::Unknown(i),
+                // TODO: I honestly have no fucking idea what KeyCodeYes is
+                pancurses::Input::KeyCodeYes => Event::Refresh,
+                // TODO: There seems to not be a Key::Break??
+                pancurses::Input::KeyBreak => Event::Refresh,
+                pancurses::Input::KeyDown => Event::Key(Key::Down),
+                pancurses::Input::KeyUp => Event::Key(Key::Up),
+                pancurses::Input::KeyLeft => Event::Key(Key::Left),
+                pancurses::Input::KeyRight => Event::Key(Key::Right),
+                pancurses::Input::KeyHome => Event::Key(Key::Home),
+                pancurses::Input::KeyBackspace => Event::Key(Key::Backspace),
+                // TODO: there seems to not be an F0 key for some reason?
+                pancurses::Input::KeyF0 => Event::Refresh,
+                pancurses::Input::KeyF1 => Event::Key(Key::F1),
+                pancurses::Input::KeyF2 => Event::Key(Key::F2),
+                pancurses::Input::KeyF3 => Event::Key(Key::F3),
+                pancurses::Input::KeyF4 => Event::Key(Key::F4),
+                pancurses::Input::KeyF5 => Event::Key(Key::F5),
+                pancurses::Input::KeyF6 => Event::Key(Key::F6),
+                pancurses::Input::KeyF7 => Event::Key(Key::F7),
+                pancurses::Input::KeyF8 => Event::Key(Key::F8),
+                pancurses::Input::KeyF9 => Event::Key(Key::F9),
+                pancurses::Input::KeyF10 => Event::Key(Key::F10),
+                pancurses::Input::KeyF11 => Event::Key(Key::F11),
+                pancurses::Input::KeyF12 => Event::Key(Key::F12),
+                // TODO: there seems to not be any F-keys above F15
+                // for some reason?
+                pancurses::Input::KeyF13 => Event::Refresh,
+                pancurses::Input::KeyF14 => Event::Refresh,
+                pancurses::Input::KeyF15 => Event::Refresh,
+                // TODO: not sure what those are
+                pancurses::Input::KeyDL => Event::Refresh,
+                pancurses::Input::KeyIL => Event::Refresh,
+                pancurses::Input::KeyDC => Event::Refresh,
+                pancurses::Input::KeyIC => Event::Refresh,
+                pancurses::Input::KeyEIC => Event::Refresh,
+                pancurses::Input::KeyClear => Event::Refresh,
+                pancurses::Input::KeyEOS => Event::Refresh,
+                pancurses::Input::KeyEOL => Event::Refresh,
+                pancurses::Input::KeySF => Event::Refresh,
+                pancurses::Input::KeySR => Event::Refresh,
+                pancurses::Input::KeyNPage => Event::Refresh,
+                pancurses::Input::KeyPPage => Event::Refresh,
+                // TODO: no way to handle this
+                pancurses::Input::KeySTab => Event::Refresh,
```

Comment:
For this reason, the tab key is actually handled as `Key::Tab` rather then the `\t` char.